python_examples/examples/bulb_example.py

```diff
@@ -12,5 +13,6 @@ def power_state(deviceId, state):
 }
 
 if __name__ == '__main__':
+    logging.basicConfig(filename='myapp.log', level=logging.INFO)
```

Comment:
Perhaps, rewrite as
`logger = logging.basicConfig(filename='myapp.log', level=logging.INFO)`

Reply:
Here I just configured logging in __main__. 
The package will take care of logging info to the myapp.log. Which will be created when an exception is raised.